assets/src/stories-editor/components/editor-carousel/index.js

```diff
@@ -24,74 +23,87 @@ import './edit.css';
 const TOTAL_PAGE_MARGIN = 50;
 const PAGE_BORDER = 1;
 
-class EditorCarousel extends Component {
-	translateWrapper = () => {
+const EditorCarousel = () => {
+	const { pages, currentPage, currentIndex, previousPage, nextPage, isReordering } = useSelect( ( select ) => {
+		const {
+			getBlockOrder,
+			getBlocksByClientId,
+			getAdjacentBlockClientId,
+		} = select( 'core/block-editor' );
+		const { getCurrentPage, isReordering: _isReordering } = select( 'amp/story' );
```

Comment:
Yeah, this is a common annoying problem. What's a good name for a boolean-returning function that doesn't clash with the name of the variable holding the result of said function?

Reply:
Good question. To avoid it we could also just use `select( 'amp/story' ).isReordering()`.